### PR TITLE
Practices v2 navigation and page scaffolding

### LINKED ISSUE #1679: Enable navigation between Practices v2.0 and Pathways
**What needs to be done? Why does it need to be done?**
Users will be able to switch between Pathways and Practices from the user profile. The existing sidebar navigation should be maintained for ID and ND until Vitals is fully migrated to Practices 

cc @agronomic to add mocks: 

**Additional context**
Add any other context about the task here.

### PR DESCRIPTION
**Pull request by macfarlandian**
## Description of the change

Defines a new view for Practices V2 (housed at `/practices` for now while we figure out what it's really going to be called) that shows up for TN only and renders a useless but non-broken page if you navigate to it (designs pending, etc). I'm not exactly sure what the access controls are going to look like so I may not have done that part quite right, guidance welcome.

## Type of change

- [x] New feature (non-breaking change that adds functionality)

## Related issues

> Closes #1679 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally
- [ ] E2E have been run for Lantern changes ([Steps in the Readme](https://github.com/Recidiviz/pulse-dashboard#running-e2e-tests))

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -99,6 +100,7 @@ const App = () => (
                   <ProtectedRoute path={PATHWAYS_PATHS.system} component={PageSystem} />
                   <ProtectedRoute path={PATHWAYS_PATHS.operations} component={PagePractices} />
                   <ProtectedRoute path={PATHWAYS_PATHS.methodology} component={PageMethodology} />
+                  <ProtectedRoute path={PATHWAYS_PATHS.practices} component={PagePracticesV2} />
```
Comment (jovergaag) [inline src/App.js:103]: I would not be opposed to renaming the other practices component to `PageOperations` and just call this `PagePractices` if that feels better.

File: src/core/PracticesTopBar/PracticesTopBar.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+
+import { palette, spacing } from "@recidiviz/design-system";
+import React from "react";
+import styled from "styled-components/macro";
+
+import recidivizLogo from "../../assets/static/images/Logo.svg";
```
Comment (jovergaag) [inline src/core/PracticesTopBar/PracticesTopBar.tsx:22]: Should this be in the design system?

**Review by jovergaag — APPROVED** [review]
Woot! It's happening 👷🏻

File: src/App.js (hunk 1 above)
Comment (macfarlandian) [reply src/App.js:103]: that's a good idea but maybe let's wait until we settle on a real name later this week? then have one big renaming party

File: src/core/PracticesTopBar/PracticesTopBar.tsx (hunk 2 above)
Comment (macfarlandian) [reply src/core/PracticesTopBar/PracticesTopBar.tsx:22]: good call, it already is

**Pull request merged**